src/main/core/logical_processor.h

```diff
@@ -14,46 +14,44 @@
 /*
  * Logical processor on which a WorkerPool runs Worker threads.
  */
```

Comment:
The `lps_` declarations in this file `src/main/core/logical_processor.h` are duplicated with `src/main/bindings/c/bindings.h`. Do we still need `src/main/core/logical_processor.h`?

Reply:
Yeah, I was thinking of leaving it there since it's a little nicer to still have the "hand crafted" C header file for reference; we should get compilation errors if any of the declarations don't match those in the auto-generated header file. OTOH maybe having them in both places is confusing, and I suppose could lead to trouble if somehow no C files included both versions of the header. Removed.